keyboards/binepad/candypad/keymaps/default/keymap.json

```diff
@@ -0,0 +1,33 @@
+{
+    "config": {
+        "features": {
+            "encoder_map": true
+        }
+    },
+    "encoders": [
+        [
+            {
+                "ccw": "KC_WH_U",
+                "cw": "KC_WH_D"
+            },
+            {
+                "ccw": "KC_VOLD",
+                "cw": "KC_VOLU"
+            }
+        ]
+    ],
+    "keyboard": "binepad/candypad",
+    "keymap": "default",
+    "layers": [
+        [
+            "KC_NUM", "KC_PSLS", "KC_PAST", "KC_PMNS",
+            "KC_P7",  "KC_P8",   "KC_P9",   "KC_PPLS",
+            "KC_P4",  "KC_P5",   "KC_P6",
+            "KC_P1",  "KC_P2",   "KC_P3",   "KC_PENT",
+            "KC_P0",             "KC_PDOT",
+                                 "MO(1)",   "KC_MUTE"
```

Comment:
```suggestion
                                 "MO(1)",   "KC_MUTE",
            "KC_NUM", "KC_PSLS", "KC_PAST", "KC_PMNS",
            "KC_P7",  "KC_P8",   "KC_P9",   "KC_PPLS",
            "KC_P4",  "KC_P5",   "KC_P6",
            "KC_P1",  "KC_P2",   "KC_P3",   "KC_PENT",
            "KC_P0",             "KC_PDOT"
```

Reply:
Thank you Z - I have committed as you wrote.